exporters/FusionRobotExporter/Source/Exporter.cpp

```diff
@@ -178,7 +178,7 @@ void Exporter::exportMeshes(BXDJ::ConfigData config, Ptr<FusionDocument> documen
 		baseJson["SoftwareExportedWith"] = "FUSION_360";
 
```

Comment:
Where does the drivetrain weight get exported?

Reply:
No where. Not in current JSON spec/engine. Looking at putting it as a joint weight for the parent node